svd-rs/CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## Unreleased
 
+- Use generic enum for types, which can be collected into SVD arrays
```

Comment:
Unnecessary comma

Reply:
The comma tells me that it is the "generic enum" that can be collected into SVD arrays, as opposed to "types". I don't know which is correct at first glance.